modules/emulator/R/GaussProcess.R

```diff
@@ -1,3 +1,9 @@
+#--------------------------------------------------------------------------------------------------#
+##'
```

Comment:
Add a short description of the function. This will translate into more informative .md files in the man folder. Specific documentation on functions will not be included in the gitbook so it's important for there to be a place where people can look to understand what a function is doing exactly.